stdlib/public/core/KeyValuePairs.swift

```diff
@@ -125,6 +125,19 @@ extension KeyValuePairs: RandomAccessCollection {
   }
 }
 
+extension KeyValuePairs {
+  
+  @available(SwiftStdlib 6.2, *)
+  public var span: Span<(Key, Value)> {
```

Comment:
Can this convert to the tuple with labels?

```suggestion
  public var span: Span<Element> {
```

Reply:
Done, though it's a slightly more hair-raising changing than just spelling it `Element`.